### PR TITLE
feat(replays):  Update onPostBuild step to generate replay onboarding docs

### PR DESCRIPTION
**Pull request by ryan953**
This is some setup so that we can implement this onboarding experience for Session Replay:

199103347-20928682-bcc3-4387-9c81-9d8a77d1aa53

I'm modeling this on the existing code for performance: https://github.com/getsentry/sentry-docs/pull/4888

What we're doing is putting the copy/instructions inside the docs repo, then pulling that into sentry as a checklist so developers can track their progress and get things setup smoothly!

Followup PRs for this will probably include creating more md files, one for each JS framework, so that the instructions are 100% copy+paste ready for any JS project.

See also: https://docs.sentry.io/contributing/onboarding-wizard/

### CODE REVIEW

File: src/wizard/javascript/replay-onboarding/javascript/2.configure.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Sentry.init({
+  dsn: "”https://examplePublicKey@o0.ingest.sentry.io/0",
+  integrations: [
+    new Replay({
+      // Capture 10% of all sessions
+      sessionSampleRate: 0.1,
+
+      // Of the remaining 90% of sessions, if an error happens start capturing
+      errorSampleRate: 1.0,
```
Comment (ryan953) [inline src/wizard/javascript/replay-onboarding/javascript/2.configure.md:24]: These are the new config fields that we're offering, as of https://github.com/getsentry/sentry-replay/pull/268

**Review by dashed — APPROVED** [review]

**Comment by AbhiPrasad** [comment]
Wait - wasn't this considered experimental @dashed? Are we fine with this approach moving forward?

**Comment by dashed** [comment]
@AbhiPrasad Ah. Did you have something else in mind? I might not have a lot of context then if there's something else in the works. 👀

**Review by AbhiPrasad — APPROVED** [review]
> @AbhiPrasad Ah. Did you have something else in mind? I might not have a lot of context then if there's something else in the works. 👀

Not sure, I have no horse in this race - maybe @lizokm can help us figure this out?

Though in the mean time, I'm fine with shipping this so that we can help unblock Ryan and Replay team!

**Comment by lizokm** [comment]
@dashed @AbhiPrasad I'd love to get a little bit more context here. Could we meet next week so that you could walk me through what's being proposed? Would really appreciate it!

**Comment by dashed** [comment]
@lizokm We can give some historical context on this sometime next week. 👍

**Pull request merged**